pkg/server/admin.go

```diff
@@ -3209,7 +3209,7 @@ func (c *adminPrivilegeChecker) hasRoleOption(
 	row, err := c.ie.QueryRowEx(
 		ctx, "check-role-option", nil, /* txn */
 		sessiondata.InternalExecutorOverride{User: user},
-		"SELECT crdb_internal.has_role_option($1)", roleOption.String())
+		"SELECT crdb_internal.has_role_option($1)", roleoption.ToString[roleOption])
```

Comment:
nit: I'd prefer if we added a method to `RoleOption` called ToString() instead of using the map directly